packages/agents/src/tests-d/use-agent-stub.test-d.ts

```diff
@@ -0,0 +1,43 @@
+import type { Agent } from "../..";
+import type { env } from "cloudflare:workers";
+import { useAgent } from "../react";
+
+declare class A extends Agent<typeof env, {}> {
+  prop: string;
+  f1: () => number;
+  f2: (a: string) => void;
+  f3: (a: number, b: string) => Promise<string>;
+  f4: (a?: string) => void;
+  f5: (a: string | undefined) => void;
+  f6: () => Promise<void>;
+}
+
+const { stub } = useAgent<A, {}>({
```

Comment:
love this idea. how about instead of calling this "stub", we call it rpc. so it'll be agent.rpc.f1()

Reply:
Sure, that even was the initial name but I changed it before submitting the PR 😅 I'll do it at the end of the day

Reply:
well, the nice thing about calling it "stub" is that's what we call it eslewhere in workers already. so maybe we it's fine to call it stub here. lemme review and land this!